spec/policies/course_policy_spec.rb

```diff
@@ -27,6 +27,10 @@
       it 'allows the :show action' do
         expect(subject.show?).to be_truthy
       end
+
+      it 'allows the :update action' do
+        expect(subject.update?).to be_truthy
+      end
     end
 
     context 'when accessing a course of a provider that the user does not have' do
```

Comment:
Just realised, there are custom matchers to test policies:

https://github.com/varvet/pundit#policy-specs